www/js/tripconfirm/trip-confirm-services.js

```diff
@@ -1,4 +1,4 @@
-angular.module('emission.tripconfirm.services', ['ionic', "emission.plugin.logger"])
```

Comment:
why is this rename critical?

Reply:
It just when we started to discussing refactoring so I think my mind started to refactor as well.